README.md

```diff
@@ -173,9 +173,13 @@ Confirm that the data has indeed been migrated by checking pgAdmin 4.
 
 ## Running Dockerized Flask App
 
+- Prerequisite: Setup docker and docker desktop (optional). Use the following link if you have an M1 Mac (https://docs.docker.com/docker-for-mac/apple-silicon/)
+
 - `cd` into the root of this repo
 - Create a database dump and save it to `docker_postgres_dump.sql` using `pg_dump --create -h localhost -U <USERNAME> whiteclaw -f docker_postgres_dump.sql`
-- Run a cluster of containers using: `docker-compose -f docker-compose-local.yml up`. This will start a Flask app that's accessible via `localhost:5000` and a PostgreSQL instance that accessible to the Flask app.
+- Make sure you have the appropriate `.env` file at the root of this repo `set -o allexport; source .env; set +o allexport`
```

Comment:
Should this link to the `.env` file format at the top of the README? If people are just reading this section, they might not what an appropriate `.env` file is

Reply:
how u do dis